src/realm/impl/sequential_getter.hpp

```diff
@@ -84,7 +89,10 @@ class SequentialGetter : public SequentialGetterBase {
 #pragma warning(push)
 #pragma warning(disable : 4800) // Disable the Microsoft warning about bool performance issue.
 #endif
+        return m_column->get(index);
 
+        // FIXME: Below optimization is skipped because array leafs might relocate during the lifetime of the 
+        // object that owns this SequentialGetter. Enable again when we have proper support for that.
```

Comment:
@rrrlasse 
won't this give a warning of unreachable code?
I suggest to "#if 0" it out.